composites/Plugin/Shared/components/HelpText.js

```diff
@@ -1,30 +1,42 @@
 /* External dependencies */
-import React from "react";
+import React, { PureComponent } from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
 
+/* Internal dependencies */
+import colors from "../../../../style-guide/colors";
+
 const YoastHelpText = styled.p`
-	font-size: 13px;
+	textColor: ${ props => props.textColor };
+	font-size: ${ props => props.textFontSize };
+	margin-top: 0;
+	
+	a {
+		color: ${ colors.$color_snippet_focus };
```

Comment:
This point still needs an answer. @hedgefield seems to me we should let links inherit the default WordPress colors for links (default: `#0073aa`), as we're already doing for all the links in the meta box. Why we should use a different color?

Reply:
Ahh that's the default link color, I thought it was the Yoast Blue (the hex code is very similar), which isn't in our colors scss, so I changed it to that. But yes if that's the link color we use everywhere, we should use that.